browser/speedreader/speedreader_tab_helper.h

```diff
@@ -109,6 +109,8 @@ class SpeedreaderTabHelper
   void SetContentStyle(ContentStyle style);
   ContentStyle GetContentStyle();
 
+  std::string GetDomain();
```

Comment:
We should name this to `GetCurrentSiteURL` - match similar to what Shields controller is doing.